lib/components/HeroCard/HeroCard.tsx

```diff
@@ -23,36 +23,36 @@ export const ColorCard = ({ title, content, actionContent, backgroundColor = '#4
       />
     ) : (
       <a href={href} className="flex rounded-[28px] outline-none transition-transform hover:scale-105 focus:scale-105">
-        <BaseCard title={title} content={content} backgroundColor={backgroundColor} />
+        <BaseCard title={title} content={content} backgroundColor={backgroundColor} href={href} />
```

Comment:
It seems weird that both the parent and child get the same prop, `href`. 
Of course the component is not using the `href` in the other situation when looking deeper, but little confusing when reading.

We can live with this and maybe conditional prop just for this is not really needed (if actionContent --> href is not then required) for now. Maybe we can refactor this later if it bothers too much.

Reply:
True and I agree.